libs/agno/agno/utils/models/aws_claude.py

```diff
@@ -152,7 +152,8 @@ def format_messages(messages: List[Message]) -> Tuple[List[Dict[str, str]], str]
             content = []
 
             if isinstance(message.content, str) and message.content:
-                content.append(TextBlock(text=message.content, type="text"))
+                if message.content != " ":
```

Comment:
Shouldn't we rather trim the message content and not add it if it is empty? In which case does this happen? I feel like this needs to be fixed at the parsing stage rather?